.github/workflows/contracts-build-and-test.yaml

```diff
@@ -1,4 +1,4 @@
-name: Build and Test Workflow
+name: Contract Build and Test Workflow
 
 on:
   push:
```

Comment:
This needs to change to the pattern in the other workflow files to allow invocation.